src/dummy-data.json

```diff
@@ -0,0 +1,310 @@
+{
+  "history": [
+    {
+      "stationName": "add new",
+      "cases": [
+        {
+          "caseTitle": "add new",
+          "caseDetails": "null"
+        }
+      ],
+      "markScheme": [{ "text": "add new", "completed": false }]
+    },
+    {
+      "stationName": "chest pain",
```

Comment:
Given that we might want to render the this specific text, could we please have `Chest Pain` instead of `chest pain`, etc.?

Reply:
We were thinking that we might use capitalise function to do that

Reply:
Why overcomplicate things -- just write it correctly in the first place.